components/FlexibleHeader/tests/snapshot/MDCFlexibleHeaderSnapshotTests.m

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Please add unit tests for the default shadow color value.

Reply:
Done